.github/renovate.json5

```diff
@@ -0,0 +1,91 @@
+{
+  "extends": [
+    "config:recommended",
+    ":approveMajorUpdates",
+  ],
+  // submit on Monday midnight (UTC)
+  "schedule": [
+    "after 12am on Monday"
+  ],
+  "minimumReleaseAge": "14 days",
+  "vulnerabilityAlerts": {
+    "labels": [
+      "type:security"
+    ],
+    "minimumReleaseAge": "1 days"
+  },
+  "labels": ["dependencies"],
+  "dependencyDashboardLabels": ["type: process"],
+  "dockerfile": {
+    "enabled": false
+  },
+  "semanticCommits": true,
+  "pip-compile": {
+    "enabled": true,
+    "fileMatch": ["(^|/)requirements\\.in$"],
+    "lockFileMaintenance": {
```

Comment:
This is the default, and can safely be removed.

Reply:
Looking at the [version] you composed for BoA, I see the difference is only in the `lockFileMaintenance` field.
Should it be removed in both locations?

[version]: https://github.com/GoogleCloudPlatform/bank-of-anthos/blob/5c9539ea0b9e4e1e6830fcb64ad4b27429630087/.github/renovate.json5#L9-L12

Reply:
apologies, i was unclear. The `lockFileMaintenance` stanza should be removed here. it is not present in the BoA variant.